README.md

```diff
@@ -47,14 +47,15 @@ We have provided unit tests for you to run. As a reminder, a few of these unit t
 To run the tests, in the command line, navigate to the project root and then run:
 
 ```bash
-$ yarn test test/adagrams.test.js
+$ npm test test/adagrams.test.js
 ```
 
 After the tests run, there is a short table that summarizes the coverage for each file. You can view complete coverage details by running:
 
 ```bash
-$ yarn run coverage
+$ npm run coverage
 ```
+(Note: This command **must** include `run` to in order to work since it is a customize script.)
```

Comment:
```suggestion
(Note: This command **must** include `run` to in order to work since it is a custom script.)
```
Do we need this note? We don't talk much (at all?) about which of the scripts are well-known or not (custom), so I don't think there's much risk of a student trying to run `npm coverage` alone. And if they did, it just wouldn't work, and they'd run the correct command, which is what we gave them.